detectionmetrics/models/torch.py

```diff
@@ -0,0 +1,206 @@
+import os
+from typing import Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from PIL import Image
+import torch
+from torch.utils.data import DataLoader, Dataset
+from torchvision.transforms import v2
```

Comment:
Reading the code below, I find this alias `v2` a little strange.

Reply:
Fixed, we now import `v2` as `transforms`, which is more self-explanatory and resembles classic torchvision usage